src/h5p-vocabulary-drill.tsx

```diff
@@ -29,21 +29,24 @@ class VocabularyDrillContentType
 
     // TODO: Translate
     const title = this.extras?.metadata.title ?? 'Vocabulary drill';
```

Comment:
Should we add a default title in semantics, or just assume that it's always set?

```suggestion
    const title = this.extras?.metadata.title ?? '';
```

Reply:
Doesn't seem possible to save the content without any title. Don't know if there are any loopholes

Reply:
Then let's just remove the fallback title